src/bench_templates/mod.rs

```diff
@@ -0,0 +1,241 @@
+use ark_crypto_primitives::sponge::{
```

Comment:
We can make this a file in `benches`, or extract it into a separate crate (ala `ark-algebra-bench-templates`)